tests/test_integrated_channels/test_integrated_channel/test_exporters/test_learner_data.py

```diff
@@ -378,6 +415,47 @@ def test_learner_data_self_paced_course(
             assert report.completed_timestamp == expected_completion
             assert report.grade == expected_grade
 
+    @mock.patch('enterprise.models.EnrollmentApiClient')
+    @mock.patch('integrated_channels.integrated_channel.exporters.learner_data.GradesApiClient')
+    @mock.patch('enterprise.api_client.discovery.CourseCatalogApiServiceClient')
+    def test_learner_assessment_data_export(
+            self,
+            mock_course_catalog_api,
+            mock_grades_api,
+            mock_enrollment_api
+    ):
+        enrollment = factories.EnterpriseCourseEnrollmentFactory(
+            enterprise_customer_user=self.enterprise_customer_user,
+            course_id=self.course_id,
+        )
+
+        mock_course_catalog_api.return_value.get_course_id.return_value = self.course_key
+
+        mock_enrollment_api.return_value.get_course_enrollment.return_value = dict(
+            mode='verified',
+        )
+
+        # Mock grades data
+        assessment_grade_data = [dict(
+            attempted=True,
+            subsection_name='subsection_1',
+            category='subsection_category',
+            percent='1.0',
```

Comment:
do these come in as strings in real life?

Reply:
Oh, percent is a float. Everything else should be ok!